.travis.yml

```diff
@@ -2,16 +2,14 @@ language: d
 sudo: false
 
 d:
+  - dmd-nightly
+  - dmd-beta
   - dmd
+  - ldc-beta
   - ldc
-  - dmd-2.069.0
-  - dmd-2.070.2
-  - dmd-2.071.2
```

Comment:
Travis complains about older platforms:

```
inifiled 1.0.1+commit.3.gda4fa76: building configuration "inifiled-test-library"...
source/inifiled.d-mixin-253(255,1): Warning: switch case fallthrough - use 'goto default;' if intended
source/inifiled.d-mixin-253(255,1): Warning: switch case fallthrough - use 'goto case;' if intended
source/inifiled.d-mixin-253(256,1): Warning: switch case fallthrough - use 'goto default;' if intended
source/inifiled.d-mixin-253(255,1): Warning: switch case fallthrough - use 'goto default;' if intended
dmd failed with exit code 1.
```

Looks like a compiler error that has been fixed in 2.072 - is there any legitimate interest in supporting outdated releases? For the build scripts at `dlang` `2.072.2` is mostly used as "stable" DMD ...